### PR TITLE
Version 1.1

### PR DESCRIPTION
**Pull request by Shashank02051997**

### CODE REVIEW

File: app/google-services.json [hunk 1]
```diff
@@ -1,3 +1,351 @@
 {
-
+  "project_info": {
```
Comment (eventers) [inline app/google-services.json]: delete this file.
In Readme.md, add this statement to configure the project with suitable google-service.json file by creating a project on Firebase

File: app/src/main/java/com/eventersapp/marketplace/ui/dashboard/ProfileFragment.kt [hunk 2]
```diff
@@ -10,32 +10,37 @@ import androidx.fragment.app.Fragment
...
 import org.kodein.di.android.x.closestKodein
 import org.kodein.di.generic.instance
 
-class ProfileFragment : Fragment(), KodeinAware {
+
+class ProfileFragment : Fragment(), KodeinAware, View.OnClickListener {
 
     companion object {
-        private const val ADDRESS = "https://goalseeker.purestake.io/algorand/testnet/account/"
+        private const val ADDRESS = "<Algorand-URL>"//Replace with Algorand address url
```
Comment (eventers) [inline app/src/main/java/com/eventersapp/marketplace/ui/dashboard/ProfileFragment.kt:26]: Add this in Readme.md, to replace purestake account

File: app/src/main/java/com/eventersapp/marketplace/util/AppConstants.kt [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+    //<editor-fold desc="Bottomsheet Tag">
+
+    const val TAG_SHOW_QR_CODE_BOTTOM_SHEET_FRAGMENT = "show_qr_code_bottom_dialog_fragment"
 
     //</editor-fold>
 
     //<editor-fold desc="Commonly used constants">
 
+    const val S3URL = "<S3-URL>"//Replace with S3 url
     const val BLOG_KEY = "<key>"//Replace with ghost blog key
```
Comment (eventers) [inline app/src/main/java/com/eventersapp/marketplace/util/AppConstants.kt:14]: Add this in Readme.md, to replace s3url and blog key

File: app/build.gradle [hunk 4]
```diff
@@ -2,37 +2,37 @@ apply plugin: 'com.android.application'
...
         DEV {
-            //applicationIdSuffix ".dev"
-            applicationId "com.eventersapp.tokengen.debug"
+            applicationId "com.eventersapp.marketplace.debug"
             dimension = 'mode'
-            resValue "string", "app_name", "Eventers Dev"
+            resValue "string", "app_name", "Eventers Marketplace Dev"
             resValue "string", "package_name", applicationId
-            buildConfigField "String", "end_point", '"<DEV-URL>"'//replace with dev url
+            buildConfigField "String", "end_point", '"https://api.eventersapp.com/v1/"'
```
Comment (eventers) [inline app/build.gradle]: remove url from code

File: app/build.gradle [hunk 5]
```diff
@@ -2,37 +2,37 @@ apply plugin: 'com.android.application'
...
         }
         PROD {
-            applicationId "com.eventersapp.consumer"
+            applicationId "com.eventersapp.marketplace"
             dimension = 'mode'
-            resValue "string", "app_name", "Eventers"
+            resValue "string", "app_name", "Eventers Marketplace"
             resValue "string", "package_name", applicationId
-            buildConfigField "String", "end_point", '"<PROD-URL>"'//replace with prod url
+            buildConfigField "String", "end_point", '"https://api.eventersapp.com/v1/"'
```
Comment (eventers) [inline app/build.gradle]: remove url from code

File: app/google-services.json (hunk 1 above)
Comment (Shashank02051997) [reply app/google-services.json]: Done

File: app/build.gradle (hunk 4 above)
Comment (Shashank02051997) [reply app/build.gradle]: Done

File: app/build.gradle (hunk 5 above)
Comment (Shashank02051997) [reply app/build.gradle]: Done

File: app/src/main/java/com/eventersapp/marketplace/ui/dashboard/ProfileFragment.kt (hunk 2 above)
Comment (Shashank02051997) [reply app/src/main/java/com/eventersapp/marketplace/ui/dashboard/ProfileFragment.kt:26]: Done

File: app/src/main/java/com/eventersapp/marketplace/util/AppConstants.kt (hunk 3 above)
Comment (Shashank02051997) [reply app/src/main/java/com/eventersapp/marketplace/util/AppConstants.kt:14]: Done

**Pull request merged**